feature/policy_forwarding/encapsulation/otg_tests/decap_static_guev1/README.md

```diff
@@ -0,0 +1,297 @@
+| RX-Flow-type#2 | IPv4 Payload        | N/A            | IPV4-SRC1        | IPV4-DST1        | 32   | 49  | N/A                                     | N/A                                          |
+| RX-Flow-type#3 | IPv6 Payload        | N/A            | IPV6-SRC1        | IPV6-DST1        | 32   | 49  | N/A                                     | N/A                                          |
+| RX-Flow-type#4 | IPv6 Payload        | N/A            | IPV6-SRC1        | IPV6-DST1        | 32   | 49  | N/A                                     | N/A                                          |
+| RX-Flow-type#7 | IP+UDP+IPv4 Payload | Outer IPGUE    | ATE-P1-Address   | ATE-P2-Address   | 35   | 68  | Varies depending on the application     | 6081 (configured non-default on DUT)         |
+|                |                     | Inner IPv4     | IPV4-SRC1        | IPV4-DST1        | 32   | 50  | N/A                                     | N/A                                          |
+| RX-Flow-type#8 | IP+UDP+IPv6 Payload | Outer IPGUE    | ATE-P1-Address   | ATE-P2-Address   | 35   | 68  | Varies depending on the application     | 6081 (configured non-default on DUT)         |
+|                |                     | Inner IPv6     | IPV6-SRC1        | IPV6-DST1        | 32   | 50  | N/A                                     | N/A                                          |
+ 
+ ### DUT Configuration
+
+1.  Interfaces: Configure all DUT ports as singleton IP interfaces towards the ATE ports
+ 
+2. GUE Decapsulation:
+    *  Configure a Loopback address with DECAP-DST_3/32 address
+    *  Configure the DUT as GUEv1 decapsulator with below fields:
+       - DECAP-DST-SUBNET/30 as decapsulation subnet-range
+          - DECAP-DST/32 is one of the IP prefix from DECAP-DST-SUBNET/30
+       - Do not enable copying TTL bits from outer to inner header post decapsulation
+       - Use single UDP destination port 6080 for both IPv4 and IPv6 payload type
+    *  Configure static routes for destination IPV4-DST2 and IPV6-DST2 towards ATE Port 2
+    *  Configure the UDP decap destination port as 6081 
+
+* DECAP-DST-SUBNET/30 subnet comprises of below IP prefixes:
+   - DECAP-DST_1/32
+   - DECAP-DST_2/32
+   - DECAP-DST_3/32
+   - DECAP-DST_4/32
+
+* DUT hosted IPv4 prefixes:
+  * Loopback Address = DECAP-DST_3/32
+
+### Test Procedure
+Traffic: 
+-  Generate the applicable GUE-encapsulated flow from ATE Port 1
+     -  Sent 1000000 packets at the 10%  of the line rate.
+     -  Keep the frame size Internet Mix.
+   
+### PF-1.4.1: GUE Decapsulation of inner IPv4 traffic using default GUE UDP port 6080
+-  Push DUT configuration
+-  Initiate traffic flow type#1  
+Verification:
+-  The outer header destination IP of the traffic falls within the configured decap range (DECAP-DST-SUBNET/30)
+-  The outer header destination port of the traffic (UDP 6080) matches the default UDP decap port criteria
```

Comment:
as noted above, there is no default in the OC model for this, the configuration must specify the destination port used

Reply:
nit:  this conflicts with line 97:
https://github.com/openconfig/featureprofiles/pull/4178/files#diff-cfed542cee25fba0c142d4357accaac8d378a13886f4eb6872a62b25177be9b6R97